routes/api/v1/foods.js

```diff
@@ -113,6 +114,22 @@ function validateRequest(req) {
   })
 }
 
+router.delete('/:id', async (req, res, next) => {
+  res.setHeader("Content-Type", "application/json");
+  try {
+    const food = await Food.findOne({where: {id: req.url.slice(1)}})
+    if (!food) {
```

Comment:
I think we should return some kind of response for both of these outcomes. If a food is deleted, it should let the user know it is deleted. If a food is not found here, it should say the requested item could not be found.

Reply:
I see what you're saying. For the `404` status I went ahead and implemented the following response ` {
        "error": "The requested food item could not be found and was therefore not deleted."
      }`. The `204` status code means `No Content` and the status code is what the project spec requests, therefore I think it's important to leave the `204` status code without content in the body.

Reply:
I shortened it to to "The requested food item could not be found." That is more consistent with other endpoint responses.

Reply:
I think no content in the body is a bad user experience but fine leaving it at least for now.